Marlin/src/module/temperature.h

```diff
@@ -179,7 +179,7 @@ enum ADCSensorState : char {
 typedef struct TempInfo {
   uint16_t acc;
   int16_t raw;
-  celsius_float_t celsius;
+  celsius_t celsius;
```

Comment:
this should be float to all this make sense...

Reply:
I'll audit its use and see whether any partial degrees are stored there.

Reply:
It is. This is where the problem start. If we store only int here, we lose decimals used in pid and other places.

Reply:
You changed all the float usages of temp to Celsius_t, in every calculation.

It's totally the opposite of the intent of my PR.

Are you going to rewrite all the pid and other calculations to work without precision?

Reply:
Yes, this one has always been float and should remain so.